### DISCUSSION TITLE
transform1InputFilePath and transform2InputFilePath in EditSequenceFile MIX command

### DISCUSSION THREAD

**Discussion by valflanza** [PlusToolkit/PlusLib#834] [Q&A]
Hello.

We are working with PULS toolkit and we want to merge a DICOM file and a tracker coordinates files. We think that the best option is to use EditSequenceFile MIX but we don't know (or we don't find) the format of the transformInputFiles.

Any suggestion?

**Comment by adamrankin** [comment]
@lassoan Does editsequencefile support DICOM image format?
